mksusecd

```diff
@@ -2970,7 +2970,7 @@ sub add_modules_to_initrd
     mkdir "$tmp_dir/lib/modules/$kernel->{version}", 0755;
     mkdir "$tmp_dir/lib/modules/$kernel->{version}/initrd", 0755;
 
-    for ("loop.ko", "squashfs.ko") {
+    for ("loop.ko", "squashfs.ko", "lz4_decompress.ko") {
```

Comment:
Since this is a Perl script and it has string constants all over the place (like most scripts), I'd vote against that; this would only tear the places where those things are used apart.